homeassistant/components/rfxtrx/__init__.py

```diff
@@ -126,6 +128,9 @@ def _ensure_device(value):
         vol.Optional(CONF_COMMAND_ON): cv.byte,
         vol.Optional(CONF_COMMAND_OFF): cv.byte,
         vol.Optional(CONF_SIGNAL_REPETITIONS, default=1): cv.positive_int,
+        vol.Optional(
```

Comment:
We don't allow changing the config yaml schema since we now use a config flow for configuration of this integration.

Reply:
Fixed.  Removed the code.  Didn't realize this part was not used any more.